lib/Undefined.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+const util = require('util');
+
+/**
+ * Error for undefined parameters
+ *
+ * @extends Error
+ * @class Errors.Undefined
+ * */
+class Undefined extends Error{
+    /**
+   * @example
+   *
+   *if(!importantPassword){
+       const err = new Errors.Undefined;
+       err.setMessage(
+           'importantPassword',
+           importantPassword
+       );
+       throw err;
+   }
+   console.log(`Dont forget this importantPassword: ${importantPassword}, don't!!!`);
+   *
+   * @example
+   *
+   * ```sh
+    git/node-error-classes/example/undefined.js:14
+         throw err;
+         ^
+
+    Undefined: Expected 'importantPassword' but to be defined or filled but was undefined or empty
+         'undefined'
+
+   * ```
+   *
+   * @method Errors.Undefined.setMessage
+   * @param  {Any} variableName name of variable
+   * @param  {Any} variable      value of varible
+   * @returns {String}    compiled error message
+   */
+    setMessage(variableName, variable){
+        this.name = 'Undefined';
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected indentation of 6 space characters but found 8.](https://www.codacy.com/app/RIAEvangelist/node-error-classes/file/1939102909/issues/source?bid=3073169&fileBranchId=3104106#l42)